src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,40 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH_DATA = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new InvalidDataException("User must not be null");
+        }
+        if (user.getLogin() == null) {
+            throw new InvalidDataException("Login mustn't be null");
+        }
+        if (user.getPassword() == null) {
+            throw new InvalidDataException("Password mustn't be null");
```

Comment:
The checkstyle rule for string constants is to use 'must not' instead of 'mustn't'. Consider revising the exception message to follow this convention.